### ISSUE TITLE
Accessibility fixes: Identifier page

### ISSUE THREAD

**Issue by mariagould** [CDLUC3/ezid#543]
The following accessibility issues should be addressed on the identifier details page (identifier landing page). 

- [x] Fix orphaned form labels: "Identifier", "Identifier as URL", "Owner", "Group", "Created", "Last modified"
- [x] Fix missing fieldsets: "Status" and "Allow harvesting/indexing"
- [x] Fix Broken ARIA reference – aria-labelledby=”fieldset-publish” (target for reference does not exist)

**Comment by sfisher** [comment]
I'm not totally sure if this is referring to the landing or the basic edit page.  Like the page https://ezid-dev.cdlib.org/id/ark:/13030/c8hh6h18 or https://ezid-dev.cdlib.org/id/ark:/13030/c8hh6h18 since I find it weird that labels would be applied to non-form elements.

So the second two bullets make sense to me for the edit page, but the first point is odd since usually only form elements accept labels in my experience.  ChatGPT says "Using \<label\> for non-form elements is not a normal or recommended practice." Though I found some other pages that say you can add labels for non-form elements if you feel it's a fun thing to do.

I guess someone did it on both these pages, so I'll add *for* with the lables and associate the labels with div ids to "fix" this problem, even though this doesn't seem totally normal to me.

**Comment by adambuttrick** [comment]
Included in https://github.com/CDLUC3/ezid/releases/tag/v3.3.2.
